jobs/kpi-forecasting/kpi_forecasting/metric_hub.py

```diff
@@ -52,17 +59,32 @@ def __post_init__(self) -> None:
             "\n", "\n" + " " * 19
         )
 
+        # Add query snippets for segments
+        if self.segments:
+            segment_select_query = []
+            for alias, sql in self.segments.items():
+                segment_select_query.append(f"  {sql} AS {alias},")
```

Comment:
Is the trailing comma here unnecessary? Maybe I'm not thinking through this correctly, but I imagine you're going to get a list like:
```python3
segment_select_query = ["A as a,", "B as b"]
```
and then when you use the `join` later the resulting string will be something like:

```SQL
A as a,,
B as b,
```

Reply:
You're right, thanks for catching that. I've changed it in the most recent commit, to remove the comma from the joined string.